wallet/src/de/schildbach/wallet/ui/dashpay/work/DeriveKeyWorker.kt

```diff
@@ -0,0 +1,31 @@
+package de.schildbach.wallet.ui.dashpay.work
+
+import android.content.Context
+import androidx.work.CoroutineWorker
+import androidx.work.WorkerParameters
+import androidx.work.workDataOf
+import de.schildbach.wallet.WalletApplication
+import org.bitcoinj.crypto.KeyCrypterException
+
+class DeriveKeyWorker(context: Context, parameters: WorkerParameters)
```

Comment:
This could be placed inside of the `SendContactRequestWorker` but wanted to present how the work chaining can be used